### ISSUE TITLE
Unrecognized arguments from lightconv example

### ISSUE THREAD

**Issue by SefaZeng** [pytorch/fairseq#2858]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
I try to train a lightconv translation model from scratch. I set the args like the example docs. 
e.g. 
`--lr-shrink 1 --max-lr 0.001 --lr 1e-7 --min-lr 1e-9 `. 
But it caused an error :
`error: unrecognized arguments: --activation-dropout 0.1 --lr-shrink 1 --max-lr 0.001 --t-mult 1 --lr-period-updates 70000`
#### Code
```
python3 $code_dir/fairseq-master/fairseq_cli/train.py \
    $data \
    --ddp-backend=no_c10d \
    --save-dir $nmt_dir/checkpoints/$signature \
    --max-update 30000 --share-all-embeddings --optimizer adam \
    --adam-betas '(0.9, 0.98)' --clip-norm 0.0 --weight-decay 0.0 \
    --criterion label_smoothed_cross_entropy --label-smoothing 0.1 \
    --min-lr 1e-09 --update-freq 16 --attention-dropout 0.1 --keep-last-epochs 10 \
    --ddp-backend=no_c10d --max-tokens 3584 \
    --lr-scheduler cosine --warmup-init-lr 1e-7 --warmup-updates 10000 \
    --lr-shrink 1 --max-lr 0.001 --lr 1e-7 --min-lr 1e-9 --warmup-init-lr 1e-07 \
    --t-mult 1 --lr-period-updates 70000 \
    --arch lightconv_wmt_en_fr_big --save-dir $SAVE \
    --dropout 0.1 --attention-dropout 0.1 --weight-dropout 0.1 \
    --encoder-glu 1 --decoder-glu 1 \
    --fp16 \
    --no-progress-bar
```
   

#### What have you tried?
I try to drop these args but the bleu score is too low.
#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):1.0
 - PyTorch Version (e.g., 1.0)1.4
 - OS (e.g., Linux):linux
 - How you installed fairseq (`pip`, source): source
 - Build command you used (if compiling from source):
 - Python version:3.6
 - CUDA/cuDNN version:10.0
 - GPU models and configuration:
 - Any other relevant information:

**Comment by alexeib** [comment]
--activation-dropout is named --relu-dropout i think
for the rest, it should be working (possibly fixed yesterday).
let me know if it still doesnt work after pulling

i will look into updating the docs

**Comment by alexeib** [comment]
actually there is no --activation-dropout in the examples
